pypeclub/OpenPype
# Nuke: adding Reformat to baking mov plugin

## Brief description
Reformat node is definable in settings

## Description
- Reformat node is disabled by default and needs to be activated. 
- Added subset name filtering on baking profiles
- reformat knobconfig with defined types
- extract review condition to keep input resolution if `reformated` tag in representation

## Testing notes:
1. Start nuke in your testing workfile, make sure it is more than fullHD resolution
2. Enable `project_settings/nuke/publish/ExtractReviewDataMov/outputs/baking/reformat_node_add`
3. make sure your Extract review preset is not having reformat to fullHD resolution (project_settings/global/publish/ExtractReview/profiles/0/outputs/h264/width or height is set to 0)
4. publish render write
5. notice that extracted review mp4 is fullHD resolution rather then the original workfile resolution
